tensorflow/compiler/xrt/xrt_memory_manager.h

```diff
@@ -168,7 +173,8 @@ xla::StatusOr<T> XRTMemoryManager::Run(
     if (result_or.status().code() != error::RESOURCE_EXHAUSTED) {
       return result_or;
     }
-    TF_RETURN_IF_ERROR(TryFreeMemoryStep(&mrctx, result_or.status()));
+    TF_RETURN_IF_ERROR(
+        TryFreeMemoryStep(&mrctx, result_or.status(), allocator));
```

Comment:
might be better to add `allocator` to `MemoryReclaimContext`, wdyt?

Reply:
Because `ScopedRef` is more accessible from everywhere. But putting allocator to `MemoryReclaimContext` seems more reasonable.